ui/src/treeDataProvider/AzExtTreeItem.ts

```diff
@@ -19,7 +19,7 @@ export abstract class AzExtTreeItem implements types.AzExtTreeItem {
     public id?: string;
     public commandId?: string;
     public commandArgs?: unknown[];
-    public iconPath?: string | Uri | { light: string | Uri; dark: string | Uri };
+    public iconPath?: string | Uri | { light: string | Uri; dark: string | Uri } | ThemeIcon;
```

Comment:
nit: looks like we could change these to `types.TreeItemIconPath` instead of spelling it all out